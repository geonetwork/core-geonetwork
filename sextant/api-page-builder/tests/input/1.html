<!DOCTYPE html>
<html lang="fr_FR" xmlns:og="http://ogp.me/ns#">
    <head>
        <meta charset="utf-8" />
        <title>Cartographie - Donnees par parametre -</title>
        <link
            rel="icon"
            type="image/png"
            href="/public/_eziweb/favicon/favicon-16x16.png"
            sizes="16x16"
        />
        <link rel="manifest" href="/public/_eziweb/favicon/manifest.json" />
        <link rel="shortcut icon" href="/public/_eziweb/favicon/favicon.ico" />
        <link
            rel="stylesheet"
            href="/public/css/f1c062ed956a9031e991060899c83516.css"
        />
        <script src="/public/js/b8ded9951e167b827e26937c4de730af.js"></script>
    </head>
    <body data-ez-root="/surval">
        <div class="cpnt c_area " id="root">
            <header class="cpnt c_area " id="header">
                <div class="c_body text-left">
                    <div class="inline">
                        <a href="/surval/"
                            ><img
                                src="/var/storage/images/_aliases/logo_main/medias-ifremer/medias-surval/logo-jumelles/1608886-4-fre-FR/Logo-jumelles.png"
                                class="img-responsive"
                                alt="Retour à l&#039;accueil"
                        /></a>
                    </div>
                </div>
                <div class="frame">
                    <ul role="tablist">
                        <li><a href="/surval/Conseils">Conseils</a></li>
                    </ul>
                </div>
            </header>
            <article class="cpnt c_showlocation fullscreen" id="body">
                <div class="embed-responsive">
                    <div
                        ng-app="gn_search_sextant"
                        ng-controller="GnCatController"
                        class="fullscreen gn"
                        sxt-size="auto"
                        ng-include="../../catalog/views/sextant/templates/index.html"
                        sxt-size-diff="50"
                    ></div>
                </div>
            </article>
            <footer class="cpnt c_area hidden-print" id="footer">
                <div class="c_body">
                    <div class="tool t_sitemaplink ">
                        <a href="/surval/sitemap">Plan du site</a>
                    </div>
                    <div class="tool t_link ">
                        <a href="/institut/Accessibilite">Accessibilité</a>
                    </div>
                    <div class="tool t_link ">
                        <a href="/institut/Mentions-legales"
                            >Mentions légales / Crédits</a
                        >
                    </div>
                </div>
            </footer>
        </div>
        <script src="/public/js/5ac30465bae6f83c6804de6330257cd4.js"></script>
        <script src="https://sextant.ifremer.fr/geonetwork/static/lib_nojquery.js"></script>
        <script src="https://sextant.ifremer.fr/geonetwork/static/gn_search_sextant.js"></script>
    </body>
</html>
