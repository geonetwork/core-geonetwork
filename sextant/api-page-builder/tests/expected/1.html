<!DOCTYPE html>
<html lang="fr_FR" xmlns:og="http://ogp.me/ns#"><head>
        <meta charset="utf-8"/>
        <title>Cartographie - Donnees par parametre -</title>
        <link href="http://mysite.org/public/_eziweb/favicon/favicon-16x16.png" rel="icon" sizes="16x16" type="image/png"/>
        <link href="http://mysite.org/public/_eziweb/favicon/manifest.json" rel="manifest"/>
        <link href="http://mysite.org/public/_eziweb/favicon/favicon.ico" rel="shortcut icon"/>
        <link href="http://mysite.org/public/css/f1c062ed956a9031e991060899c83516.css" rel="stylesheet"/>
        <script src="http://mysite.org/public/js/b8ded9951e167b827e26937c4de730af.js"></script>
    </head>
    <body data-ez-root="/surval">
        <div class="cpnt c_area" id="root">
            <header class="cpnt c_area" id="header">
                <div class="c_body text-left">
                    <div class="inline">
                        <a href="http://mysite.org/surval/"><img alt="Retour à l'accueil" class="img-responsive" src="http://mysite.org/var/storage/images/_aliases/logo_main/medias-ifremer/medias-surval/logo-jumelles/1608886-4-fre-FR/Logo-jumelles.png"/></a>
                    </div>
                </div>
                <div class="frame">
                    <ul role="tablist">
                        <li><a href="http://mysite.org/surval/Conseils">Conseils</a></li>
                    </ul>
                </div>
            </header>
            <article class="cpnt c_showlocation fullscreen" id="body">
                <div class="embed-responsive">
                    <div class="fullscreen gn" ng-app="gn_search_sextant" ng-controller="GnCatController" ng-include="../../catalog/views/sextant/templates/index.html" sxt-size="auto" sxt-size-diff="50"></div>
                </div>
            </article>
            <footer class="cpnt c_area hidden-print" id="footer">
                <div class="c_body">
                    <div class="tool t_sitemaplink">
                        <a href="http://mysite.org/surval/sitemap">Plan du site</a>
                    </div>
                    <div class="tool t_link">
                        <a href="http://mysite.org/institut/Accessibilite">Accessibilité</a>
                    </div>
                    <div class="tool t_link">
                        <a href="http://mysite.org/institut/Mentions-legales">Mentions légales / Crédits</a>
                    </div>
                </div>
            </footer>
        </div>
        <script src="http://mysite.org/public/js/5ac30465bae6f83c6804de6330257cd4.js"></script>
        <script src="http://localhost:8080/geonetwork/static/lib_nojquery.js"></script>
        <script src="http://localhost:8080/geonetwork/static/gn_search_sextant.js"></script>


</body></html>
